src/libraries/Native/build-native.cmd

```diff
@@ -120,11 +120,10 @@ echo %MSBUILD_EMPTY_PROJECT_CONTENT% > "%__artifactsDir%\obj\native\Directory.Bu
 
 if exist "%VSINSTALLDIR%DIA SDK" goto GenVSSolution
 echo Error: DIA SDK is missing at "%VSINSTALLDIR%DIA SDK". ^
```

Comment:
DIA SDK is only needed for the dactablegen tool that is specific to CoreCLR build. These DIA SDK checks can be deleted from the non-CoreCLR parts of the build.

Reply:
It would be useful if "all" relevant prerequisite checks would also be done by any wrapper scripts (like the root build.cmd) so you don't get partway into a build only to find a sub-component is missing a prereq.

Reply:
Agreed with both, merging in the spirit of incremental improvement, tracking the rest in #1251